components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -3268,6 +3268,186 @@ paths:
           source: 'curl -k -X PUT -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
           -F file=@theme.zip "https://127.0.0.1:9443/api/am/admin/v4/tenant-theme"'
 
+  ######################################################
+  # Organization theme of Devportal related APIs
+  ######################################################
+  /org-themes:
+    get:
+      operationId: getOrgThemes
+      summary: Retrieve UUIDs of org-theme contents
+      description: Returns the UUIDs of org-theme contents and their publish status.
+      parameters:
+        - name: publish
```

Comment:
Rename this as status

Reply:
Since, earlier it was decided to go with this design, I will start another discussion regarding this after alpha (Since alpha is urgent)